fe/fe-core/src/main/java/com/starrocks/catalog/BrokerMgr.java

```diff
@@ -51,7 +51,7 @@
  */
 public class BrokerMgr {
     public static final ImmutableList<String> BROKER_PROC_NODE_TITLE_NAMES = new ImmutableList.Builder<String>()
-            .add("Name").add("IP").add("Port").add("Alive")
+            .add("Name").add("IP/FQDN").add("Port").add("Alive")
```

Comment:
```
add("Host")
```
FQDN is a host.
If you change the name, you should pay attention to StarRocksManager about the usage.